packages/react/src/schematics/stories/stories.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  chain,
+  Rule,
+  schematic,
+  SchematicContext,
+  Tree
+} from '@angular-devkit/schematics';
+import { getProjectConfig } from '@nrwl/workspace';
+import { join } from 'path';
+import { CreateComponentStoriesFileSchema } from '../component-story/component-story';
+import { CreateComponentSpecFileSchema } from '../component-cypress-spec/component-cypress-spec';
+
+export interface StorybookStoriesSchema {
+  project: string;
+  generateCypressSpecs: boolean;
+  js?: boolean;
+}
+
+export function createAllStories(
+  projectName: string,
+  generateCypressSpecs: boolean,
+  js
+): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    const projectSrcRoot = getProjectConfig(tree, projectName).sourceRoot;
+    const libPath = join(projectSrcRoot, '/lib');
+
+    let componentPaths: string[] = [];
+    tree.getDir(libPath).visit(filePath => {
+      if (
+        (filePath.endsWith('.tsx') && !filePath.endsWith('.spec.tsx')) ||
+        (filePath.endsWith('.js') && !filePath.endsWith('.spec.js')) ||
+        (filePath.endsWith('.jsx') && !filePath.endsWith('.spec.jsx'))
+      ) {
+        componentPaths.push(filePath);
+      }
+    });
+
+    return chain(
+      componentPaths.map(componentPath => {
+        const relativeCmpDir = componentPath.replace(
+          join('/', projectSrcRoot, '/'),
+          ''
+        );
+
+        return chain([
+          schematic<CreateComponentStoriesFileSchema>('component-story', {
+            componentPath: relativeCmpDir,
```

Comment:
Would there be any problems with a `.js` file that doesn't contain any React components?

Reply:
fixed